src/Test/L0/Worker/StepsRunnerL0.cs

```diff
@@ -304,13 +305,13 @@ private TestHostContext CreateTestContext([CallerMemberName] String testName = "
                 // Arrange.
                 var variableSets = new[]
                 {
-                    new[] { CreateStep(TaskResult.Failed), CreateStep(TaskResult.Succeeded) },
-                    new[] { CreateStep(TaskResult.Failed), CreateStep(TaskResult.Succeeded, continueOnError: true) },
-                    new[] { CreateStep(TaskResult.Failed), CreateStep(TaskResult.Succeeded, critical: true) },
```

Comment:
It didn't make sense to set a Result when you expect the value to reflect the result of the operation. So removed it.